spec/requests/api/v1/ecf_participants_spec.rb

```diff
@@ -200,6 +200,26 @@
           end
         end
 
+        context "when cohort parameter is supplied" do
+          it "returns participants only within that cohort" do
+            next_cohort = create(:cohort, :next)
```

Comment:
Chatted to @StupidCodeFactory in person but for posterity: I'll revisit this test setup after https://github.com/DFE-Digital/early-careers-framework/pull/2225 is merged to simplify and improve using the new factories